### PR TITLE
Alerting: Update configuration.md and notification_examples.md docs

### PR DESCRIPTION
**Pull request by ivanahuckova**
**content/docs/alerting/configuration.md**
- When downloading Alertmanager, it contains `alertmanager.yml` file and not `simple.yml`.

**content/docs/alerting/notification_examples.md**
- Listing very useful resource that I have found accidentally. I think that slack notification exmaples is a perfect place for this kind of resource.

### CODE REVIEW

File: content/docs/alerting/notification_examples.md [hunk 1]
```diff
@@ -100,4 +100,4 @@ templates:
 - '/etc/alertmanager/templates/myorg.tmpl'
 ```
 
-This example is explained in further detail in this [blogpost](https://prometheus.io/blog/2016/03/03/custom-alertmanager-templates/).
+This example is explained in further detail in this [blogpost](https://prometheus.io/blog/2016/03/03/custom-alertmanager-templates/). To test and iterate on your Prometheus Alertmanager notification templates for Slack you can use following [tool](https://juliusv.com/promslack/).
```
Comment (brian-brazil) [inline content/docs/alerting/notification_examples.md]: Integrations go on that page, not as an offhand mention in the docs on notification templates generally.

File: content/blog/2016-03-03-custom-alertmanager-templates.md [hunk 2]
```diff
@@ -124,3 +124,5 @@ templates:
 
 We reload our Alertmanager by sending a `SIGHUP` or restart it to load the
 changed configuration and the new template file. Done.
+
+To test and iterate on your Prometheus Alertmanager notification templates for Slack you can use following [tool](https://juliusv.com/promslack/).
```
Comment (brian-brazil) [inline content/blog/2016-03-03-custom-alertmanager-templates.md]: use the

Comment (ivanahuckova) [inline content/blog/2016-03-03-custom-alertmanager-templates.md]:
```suggestion
To test and iterate on your Prometheus Alertmanager notification templates for Slack you can use the following [tool](https://juliusv.com/promslack/).
```

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**